src/commonMain/kotlin/exchange.dydx.abacus/utils/Constants.kt

```diff
@@ -26,3 +26,13 @@ internal const val GEO_POLLING_DURATION_SECONDS = 10.0
 
 // Autosweep Constants
 internal const val MIN_USDC_AMOUNT_FOR_AUTO_SWEEP = 20000
+
+internal val OSMOSIS_SWAP_VENUE = mapOf(
```

Comment:
Nit:  Prefer keeping those constants in OnboardingSupervisor.kt as private vals, since they are only referenced there.